src/sentry/spans/consumers/process/factory.py

```diff
@@ -25,6 +27,12 @@ def _deserialize_span(value: bytes) -> Mapping[str, Any]:
 
 
 def process_message(message: Message[KafkaPayload]):
+    if not options.get("standalone-spans.process-spans-consumer.enable"):
+        return
+
+    if random.random() >= options.get("standalone-spans.process-spans-consumer.rollout"):
```

Comment:
we need all the spans in a project, this one is just for initial rollout when we test out a single project

Reply:
I don't think you need this one. If 1 project breaks the rollout, we just proved this approach is not viable. Otherwise, we can scale.